### PR TITLE
readme indexing and some formatting

### PR DESCRIPTION
**Pull request by vabhishek-me**
I've added some indexing to readme for people to have a quick look at all the resources. Hope this helps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+### [ReactJS ](./ReactJS.md)
+
+### [Redux](./Redux.md)
+
+### [Python](./Python.md)
+
+### [Django - Python Framework](./Django.md)
+
+
+
```
Comment (harshitjuneja) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+  ```markdown
+  # [Topic Name] Related Resources
+
+  ## For Beginners
+
+  ## For People with Intermediate Level Expertise
+
+  ## For Experts
+  ```
+
```
Comment (harshitjuneja) [inline README.md]: The link to the template is sufficient. The template can evolve over time, duplicating it here will be a maintenance burden in the future.

**Review by harshitjuneja — COMMENTED** [review]
1. Nice thoughts putting in references to topic files. 
Every new topic will need to be added but maintenance would be worth it. 

2. Template could evolve over time, duplicating it's content here doesn't add much value

Work on #2 and update the pull request.

**Comment by vabhishek-me** [comment]
Now ready to merge.....

**Comment by saranshkataria** [comment]
Would this not end up being too long a list once resources are added for more topics? Should we categorize them into folders according to areas (web dev, AI etc.)? I was putting off that discussion but since you brought up categorization I think we need to think about that as well

**Comment by vabhishek-me** [comment]
Let's keep this indexing like it is for now..... 
Once it becomes large, I'll find out some way to index it better.

**Comment by saranshkataria** [comment]
@harshitjuneja thoughts?

**Comment by saranshkataria** [comment]
I think we can split the readme file into sections like web dev, underneath it have tables with column values as technologies?

**Comment by harshitjuneja** [comment]
Yes, let's go with what's suggested by @saranshkataria. Tabular format is scalable. @vabhishek-me thoughts ?

**Comment by vabhishek-me** [comment]
Ya @harshitjuneja tabular format will work better..... what columns do you need?

Resource Name? and?

**Comment by saranshkataria** [comment]
@vabhishek-me that is it, there will be multiple columns in a single row (maybe 4-5 per row) with resource name as cells

**Comment by vabhishek-me** [comment]
I didn't get that..... can you make a sample like

|something|something|something
|-----|----|---|
|nothing|nothing|nothing|

**Comment by saranshkataria** [comment]
heading (web dev for eg)

topic | topic | topic
-- | -- | --
topic | topic | topic

Topics could be html, js etc etc

**Comment by vabhishek-me** [comment]
@saranshkataria I tried using the table format but it doesn't look good..... and also categorization and searching becomes tough...... Like for example, where would you put django? backend? webdev? api dev?

I'll say let's go with the one line format and decrease the text size (we can use bullet points below subheadings).

This will be more beneficial in my opinion.

**Comment by vabhishek-me** [comment]
I propose this..... @harshitjuneja @saranshkataria have a look

**Comment by saranshkataria** [comment]
This looks okay for now. But will be a problem in the near future. Let's think of a better way for that. Merging this in the meantime.

**Pull request merged**